src/bot.py

```diff
@@ -31,7 +32,6 @@ class BotConfigData:
 
 def configure() -> BotConfigData | None:
     if not os.path.exists(configFileName):
-        with open(f"{configFileName}.example", "r") as example, open(configFileName, "w") as configFile:
```

Comment:
Did you delete this line by mistake? The next line will crash now as the variables 'example' and 'configFile' don't exist

Reply:
yes. honest mistake cause i'm unfamiliar with the github desktop GUI. Fixing